scout/parse/variant/transcript.py

```diff
@@ -163,59 +117,8 @@ def parse_transcripts(raw_transcripts, allele=None):
         if entry.get("GNOMAD_MT_AF_HET", "") != "":
             transcript["gnomad_mt_heteroplasmic"] = float(entry.get("GNOMAD_MT_AF_HET"))
 
-        # Check frequencies
-        # There are different keys for different versions of VEP
-        # We only support version 90+
-        thousandg_freqs = []
-        gnomad_freqs = []
-        try:
-            # The keys for VEP v90+:
-            # 'AF' or '1000GAF' - 1000G all populations combined
-            # 'xxx_AF' - 1000G (or NHLBI-ESP) individual populations
-            # 'gnomAD_AF' - gnomAD exomes, all populations combined
-            # 'gnomAD_xxx_AF' - gnomAD exomes, individual populations
-            # 'MAX_AF' - Max of all populations (1000G, gnomAD exomes, ESP)
-            # https://www.ensembl.org/info/docs/tools/vep/vep_formats.html
-
-            # Loop over all keys to find frequency entries
-            for key in entry:
-                # All frequencies endswith AF
-                if not key.endswith("AF"):
-                    continue
-
-                value = entry[key]
-                if not value:
-                    continue
-
-                # This is the 1000G max af information
-                if key == "AF" or key == "1000GAF":
-                    transcript["thousand_g_maf"] = float(value)
-                    continue
-
-                if key == "GNOMAD_AF":
-                    transcript["gnomad_maf"] = float(value)
-                    continue
-
-                if key == "EXAC_MAX_AF":
-                    transcript["exac_max"] = float(value)
-                    transcript["exac_maf"] = float(value)
-                    continue
-
-                if "GNOMAD" in key:
-                    gnomad_freqs.append(float(value))
-
-                else:
-                    thousandg_freqs.append(float(value))
-
-            if thousandg_freqs:
-                transcript["thousandg_max"] = max(thousandg_freqs)
-
-            if gnomad_freqs:
-                transcript["gnomad_max"] = max(gnomad_freqs)
-
-        except Exception as err:
-            LOG.debug("Something went wrong when parsing frequencies")
-            LOG.debug("Only splitted and normalised VEP v90+ is supported")
+        # Update transcript with maximum frequencies found in entry
```

Comment:
update docstring or function name instead of adding comment!

Reply:
Done!